From afe274d25a11ca480740d2243e57755e7395e1d2 Mon Sep 17 00:00:00 2001
From: Uplink Bot <uplink@company.example>
Date: Wed, 16 Sep 2026 22:50:54 +0000
Subject: [PATCH] fix: sync workflow

Internal fix of sync workflows

----- Uplink: internal below this line -----

Uplink-Patch-Id: upl_51db40f32a
---
 .github/workflows/uplink-sync.yml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.github/workflows/uplink-sync.yml b/.github/workflows/uplink-sync.yml
index 413f3fff2761d0a660eebd7d303a2223a1a9a21b..52a851c59d0bbf1e560a38d3b10275f072e6737e 100644
--- a/.github/workflows/uplink-sync.yml
+++ b/.github/workflows/uplink-sync.yml
@@ -33,6 +33,7 @@ jobs:
       - name: Sync with upstream
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          UPLINK_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           git fetch origin '+refs/heads/uplink/state:refs/heads/uplink/state'
           main_before=$(git rev-parse HEAD)
-- 
2.55.0
